company/okrs/2020_q1.md

```diff
@@ -0,0 +1,86 @@
+   1. Sales KR: Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Sales KR: Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
+   1. VP Business (Success) KR: Generate at least [$M][M] in net new ARR from existing accounts.
+   1. VP Business (Success) KR: Hire a Head of Customer Success who can own this function.
+   1. VP Business (Success) KR: Reach monthly NPS of 60 at all Tier 1 customers.
+
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: Deliver on product roadmap**
+   1. Product KR: Roadmap communicates rough timeline for upcoming work.
+   1. Product KR: Each feature is reviewed before release to ensure quality UX.
+1. **Product: Reduce product friction to accelerate customers through pipeline**
+   1. Product KR: Shorten onboarding time of "intent to try Sourcegraph" to "whole organization is using Sourcegraph".
+   1. Product KR: Shorten product feedback cycle on new features.
+1. **Product: Build the product and design team to support engineering and ARR growth**
+   1. Product KR: Grow team to plan (+1 UX Designer, +1 PM).
+   1. Product KR: Handbook describes product strategy and processes.
+   1. Product KR: Customer feature prioritization process scales with team and customer growth.
+
+Build out vision for porduct and run customer validation discussions
+
+## Engineering
+
+1. **VP Eng: Build a world class engineering team.**
+   1. VP Eng KR: Grow team to plan (+1 code intel, +2 distribution, +2 web, +1 core services).
+   1. VP Eng KR: Transition an existing teammate to be (or hire) an Engineering Manager of the distribution team.
+   1. VP Eng KR: Document how we do performance reviews and determine compensation for teammates in engineering roles.
+1. **VP Eng: Create a data driven planning process.**
+   1. VP Eng KR: Each item in a tracking issue has an estimate of time or size.
+   1. VP Eng KR: Each team commits to a written work plan for each release cycle that accounts for the work estimates and the team's capacity.
+   1. VP Eng KR: Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+1. **Distribution: Resolve issues for critical customer deals and expansions**
+1. **Distribution: Improve developer productivity**
+   1. Distribution KR: Lower build times to X
```

Comment:
@beyang I think we have reached diminishing marginal returns on build times. I think a different metric that would be more impactful would be time it take for a line of code to get deployed to sourcegraph.com after getting merged into master.

Reply:
Chiming in to add a +1 to @nicksnyder's suggestion here 😄 

Another thought I had when seeing "Improve developer productivity" as objective: who owns alerting and flaky builds? I think getting rid of the alerting flakes and reducing the e2e test flakiness would probably be the biggest developer productivity boost at the moment.

Reply:
Given the current state of alerts and e2e flakiness, and the correlation to being able to automate the release process, I think an explicit OKR around e2e test reliability would be nice.